fishy/constants.py

```diff
@@ -1,3 +1,6 @@
 apiversion = 2
 chalutier = ("Chalutier", "https://github.com/fishyboteso/Chalutier/raw/619b4ab0b8ff91746afda855542e886d27b7a794/Chalutier_1.1.2.zip", 112)
 lam2 = ("LibAddonMenu-2.0", "https://www.esoui.com/downloads/dl7/LibAddonMenu-2.0r32.zip", 32)
+
+fishyqr = ("FishyQR", "https://github.com/fishyboteso/FishyQR/files/6329586/FishyQR.zip", 1)
```

Comment:
for sake of sanity, can you please use the correct versions? 100 for fishyqr and 30 for libgps